[HANDLOWY] Modyfikacja BD oraz modelu w VS

Maciej44Kowalski opened the pull request.

Maciej44Kowalski committed:
[HANDLOWY] Modyfikacja BD oraz modelu w VS

rudnickirobert commented on RoofingCompany.sql:
```diff
@@ -768,7 +770,7 @@ Unit.UnitName as [Jednostka],
 FROM Unit INNER JOIN (PartType INNER JOIN Part ON PartType.IdPartType = Part.IdPartType) 
 ON Unit.IdUnit = Part.IdUnit
-ORDER BY Part.PartName;
+--ORDER BY Part.PartName;
```
nie commitujemy zakomentowanego kodu

rudnickirobert commented on RoofingCompany.sql:
```diff
@@ -779,7 +781,7 @@ FROM Unit INNER JOIN (Maintenance INNER JOIN (Part INNER JOIN MaintPart
 ON Maintenance.IdMaintenance = MaintPart.IdMaintenance) 
 ON Unit.IdUnit = Part.IdUnit
-ORDER BY Maintenance.DateAcceptOrder DESC;
+--ORDER BY Maintenance.DateAcceptOrder DESC;
```
jw

rudnickirobert commented on RoofingCompanyInserts.sql:
```diff
@@ -650,7 +650,7 @@ insert into PlannedProductionEmployeeDetails (IdProces, IdEmployee, StartDate, E
 	(10,1, '2019-08-15','2019-09-30');
 
- insert into ProductionProces(IdPlan, StartDate, EndDate)
+ insert into ProductionProcess(IdPlan, StartDate, EndDate)
```
ta tabela zmienia nazwę co 5 minut- proszę przekazac grupie że prawidlowa nazwa to `ProductionProcess` i taka ma zostac

Maciej44Kowalski committed:
[Handlowy] Problem z wersją na masterze - Konflikt rozwiazywany dla brancha #83
Merge branch 'Feature/Production/New_production_plan_comboboxes' of https://github.com/rudnickirobert/KWZP2019 into feature/handlowy/dbUpdate

rudnickirobert reviewed (changes requested):
konflikty

Maciej44Kowalski committed:
[handlowy] Rozwiązanie konfliktów

Merge branch 'master' of https://github.com/rudnickirobert/KWZP2019 into feature/handlowy/dbUpdate
